cpp/benchmarks/hashing/hash.cpp

```diff
@@ -43,7 +43,7 @@ static void bench_hash(nvbench::state& state)
 
   // collect statistics
   cudf::strings_column_view input(data->get_column(1).view());
-  auto const chars_size = input.chars_size();
+  auto const chars_size = input.chars_size(cudf::get_default_stream());
```

Comment:
```suggestion
  auto const chars_size = input.chars_size(stream);
```
I think we can just use the variable declared at line 41